redis.conf

```diff
@@ -1479,23 +1479,25 @@ aof-timestamp-enabled no
 #
 # shutdown-timeout 10
 
-################################ LUA SCRIPTING  ###############################
+########################## LONG BLOCKING COMMANDS #############################
```

Comment:
I don't like the title, too many commands can block redis for a long time like `KEYS` `FLUSHALL` event `HGETALL` on a big key. But the threshold only works for FUNCTION(lua script) and modules.

Reply:
Non-deterministic blocking commands?